### PR TITLE
Single movie api

### PR DESCRIPTION
**Pull request by daomeow**
## Is this a fix or a feature?
- Fix

## What is the change?
- Moved API calls into its own file and added the single movie fetch call 

## What does it fix?
-  Update styling/responsiveness to mobile, ipad and desktop views 

## Where should the reviewer start?
- MovieDetails.js at line 1 and apiCalls.js at line 1

## How should this be tested?
- Locally

### CODE REVIEW

File: src/Components/App/App.js [hunk 1]
```diff
@@ -21,12 +22,11 @@ export default class App extends Component {
...
   componentDidMount() {
-    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
-      .then(response => response.json())
+      getMovies()
       .then(data => {
-        this.setState({movies: data.movies})
+        this.setState({ movies: data.movies })
       })
-      .catch(error => console.log(error))
+      .catch(error => this.setState({ error }))
```
Comment (tysnj) [inline src/Components/App/App.js:32]: After our discussion, we'll want to refactor this to handle the 500. We can simply let the user know something went wrong!

**Comment by tysnj** [comment]
Good work separating the concerns and pulling API calls into its own file

File: src/Components/App/App.js (hunk 1 above)
Comment (daomeow) [reply src/Components/App/App.js]: I just pushed up the update for this error handling - thanks T!

Comment (tysnj) [reply src/Components/App/App.js]: Thank you for the quick fix!

File: src/Components/MovieDetails/MovieDetails.css [hunk 2]
```diff
@@ -0,0 +1,169 @@
+/* * {
```
Comment (sertmer) [inline src/Components/MovieDetails/MovieDetails.css:1]: Love all these changes, but they don't seem to be related to refactoring API calls - I might have made these their own PR, to be honest

File: src/Components/MovieDetails/MovieDetails.js [hunk 3]
```diff
@@ -1,11 +1,13 @@
 import React from 'react';
-import movieData from '../sample-data';
 import './MovieDetails.css';
 
 const MovieDetails =({ movieInfo }) => {
-  const { poster_path, title, average_rating, release_date, backdrop_path } = movieInfo; 
+  const { poster_path, title, average_rating, release_date, backdrop_path, overview, budget, revenue, genres, tagline, runtime } = movieInfo;
```
Comment (sertmer) [inline src/Components/MovieDetails/MovieDetails.js:5]: Glad you did all this destructuring on it's own line, rather than in the props. Much more legible

**Review by sertmer — COMMENTED** [review]
Nice work yall! I left some comments - be sure to check that out!

**Pull request merged**